malevich/_templates/metascript/flow_stub/body.jinja2

```diff
@@ -2,4 +2,37 @@
 
 {% for integration in integrations -%}
     {%- include "stub.jinja2" -%}
-{%- endfor %}
\ No newline at end of file
+{%- endfor %}
+
+{% with i = integrations | first %}
+def {{reverse_id}}(
+    version: Literal['{{i.version}}'],
+    branch: Literal['{{i.branch}}'],
+    *,
+    run_deployment_id: str | None = None,
+    run_task_policy: Literal['only_use'] | Literal['use_or_new'] = 'use_or_new',
+    get_task: bool = False,
+    wait_for_results: bool = True,
+    **kwargs,
+):
+    task = Space(
```

Comment:
Code will be harshly repeated. Put all the code in the decorator, and attach decorator **to the last non-overloaded function**.

```
@overload
def flow(version: Literal, branch: Literal, args...): ...

@overload
def ...

@integration(...)
def flow(version, branch, *, ..., **flow_kwargs) -> ...:
       pass